pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -142,47 +149,97 @@ func (h *NetworkUtilsHandler) UseIptables() bool {
 
 	return true
 }
-func (h *NetworkUtilsHandler) IptablesNewChain(table, chain string) error {
-	iptablesObject, err := iptables.New()
+
+func (h *NetworkUtilsHandler) ConfigureIpv6Forwarding() error {
+	_, err := exec.Command("sysctl", "net.ipv6.conf.all.forwarding=1").CombinedOutput()
```

Comment:
Are we adding the proper policy to run sysctl at container ? 
https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#podsecuritypolicy

Reply:
Would it work if don't?

Reply:
virt-launcher has ipv4 enabled by default. But the ipv6 is not. Enabling ipv6 forwarding on the node doesn't help. Do you have another idea how we can make the ipv6 enabled on the virt-launcher?

Reply:
It is also not always the case for all ipv4 providers ( https://github.com/kubevirt/kubevirt/issues/2942). You could consider moving this part to `virt-handler` and enable it for both ipv4 and ipv6 always. virt-handler is fully privileged and can enter the right network namespace.

Reply:
This code is currently in virt-handler. I would prefer to enable the ipv4 in a separate PR.